src/fluidplayer.js

```diff
@@ -2394,12 +2394,16 @@ const fluidPlayerClass = function () {
     };
 
     self.showControlBar = () => {
+        window.addEventListener('touchstart', function onFirstTouch() {
+            window.USER_IS_TOUCHING = true;
```

Comment:
Not the best way to solve this issue by far. 
a) it adds the global variable for the whole window, we should avoid that
b) it uses touchstart event alos for the whole window..
c) this USER_IS_TOUCHING is never ever unset after it is set to true once.. Is that intended?

It's intentional that cursor is hidden after a while (it would be annoying for people while they are watching if they didn't). It should indeed reappear as the mouse is moved (together with controls indeed)

Is this code what shows suff or what hides stuff? self.domRef.player.style.cursor = 'default';
I would question the purpose of it and potentially the way the isTouchDevice function was written first and possibly target changes there, rather than trying this hack, whish seems suspicious and for sure isn't contained enough.

Reply:
@golikov The problem is with computers that are touchable at the same time. Hybrid computers. And because it's hybrid, it behaves like a cell phone and the cursor doesn't appear. This piece of code is to see if the person is using the mouse or the touch screen. If you are using the mouse then the cursor appears if you are using your finger on the screen the mouse does not appear and it behaves like a mobile device

Reply:
Thanks for the explanation. All the problematic points I mentioned are still valid.

Reply:
@golikov can you check the new changes, plase?

Reply:
Sorry still not up to standards.
I have a better suggestion how to solve it in a universal way that is more reliable and doesn't introduce permanent state to the player instance (considering that hybrid device can be used with a mouse again after touching a screen once at any moment, and that you are registering an event listener in a middle of answering another event actually).

So here's a suggestion, this is how the showControlBar is called: 
videoWrapper.addEventListener('mouseenter', self.showControlBar, false);
videoWrapper.addEventListener('touchstart', self.showControlBar, false);

Why not remove the whole check for (!self.isTouchDevice() || !isTouching) and intead check the event.type of the event that triggered this function or something like that?

Reply:
@golikov Can you check again?